src/Azure-Migrate-Export.csproj

```diff
@@ -49,6 +49,7 @@
     <Reference Include="System.Xml.Linq" />
     <Reference Include="System.Data.DataSetExtensions" />
     <Reference Include="Microsoft.CSharp" />
+	<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <Reference Include="System.Data" />

```

Comment:
I think this is an auto - change, we don't need two versions for Newtonsoft.json, please upgrade all version to 13.0.3 or keep using 13.0.3

Reply:
The package is present Vrinda, please re-verify this.